components/dashboard/src/projects/NewProject.tsx

```diff
@@ -341,7 +341,8 @@ function GitProviders(props: {
 
     useEffect(() => {
         (async () => {
-            setAuthProviders(await getGitpodService().server.getAuthProviders());
+            const providers = await getGitpodService().server.getAuthProviders();
+            setAuthProviders(providers.filter(p => ["github.com", "gitlab.com"].includes(p.host)));
```

Comment:
**question**: Does this limit Projects for repositories only from `github.com` and `gitlab.com`? I thought Projects could support repositories from self hosted GitHub and GitLab instance, but this is also something we could leave out of the scope of this first iteration. 💭

Reply:
1. The current GitHub App integration is hard coded with github.com. 
2. I'm new to the GitLab integration for prebuilds in general. It would require testing as well.

Reply:
Aha! Thanks for the insight, @AlexTugarev! Sounds good to leave this out of the scope for now until further demand. Added https://github.com/gitpod-io/gitpod/issues/5115 to track this feature request.

Reply:
This would still need an approval from a [dashboard owner](https://github.com/gitpod-io/gitpod/pull/5113#issuecomment-895112689). Looping in @jankeromnes and @svenefftinge in case any of them can take a look at this two-line change. 🏓

Reply:
Fixing this in https://github.com/gitpod-io/gitpod/pull/5192 😁 also taking a quick look!